### PR TITLE
chore(#8184): update replication performance suite to use new endpoints

### PR DESCRIPTION
**Pull request by dianabarsan**
# Description

medic/cht-core#8184

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs

If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8184-updated-performance-suite/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8184-updated-performance-suite/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:8184-updated-performance-suite/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/factories/cht/generate.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  if (submitter) {
+    context.contact = {
+      _id: submitter.contact._id,
+      parent: submitter.contact.parent,
+    };
+  }
+  return context;
+};
+
+const createData = ({ healthCenter, user, nbrClinics=10, nbrPersons=10 }) => {
```
Comment (ngaruko) [inline tests/factories/cht/generate.js:25]: wow... this is an extremely useful function 👍

**Review by ngaruko — CHANGES_REQUESTED** [review]
Very interesting work here. Extremely impressed with the generateData function. Just one comment/suggestion, if it is at all feasible.

File: tests/e2e/default/telemetry/telemetry.wdio-spec.js [hunk 2]
```diff
@@ -70,7 +70,7 @@ describe('Telemetry', () => {
     expect(telemetryEntry.doc).to.deep.nested.include({
       'metadata.year': moment().year(),
       'metadata.month': moment().month() + 1,
-      'metadata.day': moment().day() + 1,
+      'metadata.day': moment().date() - 1, // we "rolled" back time to yesterday
```
[the remark this answers is not in the document]
Comment (dianabarsan) [reply tests/e2e/default/telemetry/telemetry.wdio-spec.js:73]: This is already in `master` I think, and I "hotpatched" this branch. I'll just merge so this change doesn't get visible.

**Review by ngaruko — APPROVED** [review]
Looks good.

**Pull request merged**